concepts/05 UI Components/Lookup/00 Getting Started with Lookup/10 Customize the Drop-Down Menu.md

```diff
@@ -0,0 +1,76 @@
+On desktops and iOS devices, Lookup's drop-down menu is the [Popover](/concepts/05%20UI%20Components/Popover/00%20Overview.md '/Documentation/Guide/UI_Components/Popover/Overview/') UI component; on other devices, it is the [Popup](/concepts/05%20UI%20Components/Popup/00%20Getting%20Started%20with%20Popup '/Documentation/Guide/UI_Components/Popup/Getting_Started_with_Popup/') component.
```

Comment:
Menu customization isn't a priority feature and should be way lower in the article.

Reply:
Noted